Automattic/jetpack
# Add Jetpack Backup Daily and Real-time to Plans page

#### Changes proposed in this Pull Request:
This PR adds a card to the Plans page to sell Jetpack Backup, and also reverses the order of the plans on the page. Subsequent parts of the Jetpack Backup design will be implemented in different PRs.

Note that there are subtle visual differences between this PR and the Figma design. Obvious/easy ones can be addressed here during review, but harder things should be moved to another PR.

Current development:
<img width="560" alt="image" src="https://user-images.githubusercontent.com/42627630/68362727-e48e4680-00ed-11ea-9cd5-9f981578abf7.png">

Figma design:
<img width="589" alt="image" src="https://user-images.githubusercontent.com/42627630/68362750-f7088000-00ed-11ea-85cb-b5d3605504be.png">

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
See the P2 here for the design of this PR: p1HpG7-7MK-p2 (specifically the part titled "WP Admin Desktop")
See the P2 here for the overall MT: p1HpG7-7ET-p2

#### Testing instructions:
* Start on a Jetpack connected site with a free plan.
* Visit Dashboard > Plans
* Verify that a Jetpack Backup upgrade prompt shows at the top of the page
* Verify that toggling the radio buttons changes the upgrade button text and link
* Verify that the plans are being displayed in reverse order

#### Proposed changelog entry for your changes:
* N/A, this is a part of the much larger Jetpack Backup single product project which should probably be a single entry
